src/python_routing_v02/fast_reach/fortran_wrappers.pxd

```diff
@@ -34,5 +34,8 @@ cdef extern from "pyMCsingleSegStime_NoLoop.h":
                                   float *depthp,
                                   float *qdc,
                                   float *velc,
-                                  float *depthc) nogil;
+                                  float *depthc,
+                                  float *ck,
+                                  float *cn,
+                                  float *X) nogil;
```

Comment:
`fortran_wrappers.pxd` changed to handle additional courant returns from `pyMCsingleSegStime_NoLoop.f90` (and `pyMCsingleSegStime_NoLoop.h`)